src/lib/fetchers/callBackendToUpdateInhouseDb.ts

```diff
@@ -18,6 +20,7 @@ export async function callBackendToUpdateInhouseDb({
     },
     method: 'POST',
     mode: 'cors',
+    body: tagName
```

Comment:
suggestion (if-minor): I'd prefer this to be in a query parameter or in the URL (like `${baseUrl}/legacy/api/${osmType}/${osmIdAsNumber}/refresh/${tagName}`), but as this is just for a test and we're going to have a v3 API after all our experiments, I'd be totally okay with this API for now :)